src/averageAges.js

```diff
@@ -37,7 +59,24 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  if (withChildren === undefined) {
```

Comment:
```suggestion
  if (!withChildren) {
```
it could be like this - no need to compare with undefined
what if it will be `false` ?